isolate_example/lib/page_one.dart

```diff
@@ -84,7 +176,10 @@ class SmoothAnimationWidgetState extends State<SmoothAnimationWidget>
             decoration: BoxDecoration(
               gradient: LinearGradient(
                 begin: Alignment.topLeft,
-                colors: [Colors.blueAccent, Colors.redAccent],
+                colors: [
+                  Colors.blueAccent,
+                  Colors.redAccent
```

Comment:
```suggestion
                  Colors.redAccent,
```
Nit: Trailing commas are generally used for constructors that have more than one argument